src/view/Office.vue

```diff
@@ -303,6 +303,7 @@ export default {
 					this.loading = LOADING_STATE.FRAME_READY
 					this.$emit('update:loaded', true)
 					FilesAppIntegration.initAfterReady()
+					this.sendPostMessage('Error_Messages', { websocketconnectionfailed: 'Failed to establish socket connection or socket connection closed unexpectedly. The reverse proxy might be misconfigured, please contact the administrator. For more info on proxy conifguration please checkout https://docs.nextcloud.com/server/latest/admin_manual/office/proxy.html' })
```

Comment:
Does it make sense to apss the URL separately so it is possible to render that within online as an actual clickable link?

Reply:
cc @pedropintosilva

Reply:
@juliushaertl good idea probably best to do:

```json
{
 "MessageId" : "Error_Messages",
 "Values": {
     "websocketconnectionfailed": { msg: "Failed to establish socket connection or socket connection closed unexpectedly. The reverse proxy might be misconfigured, please contact the administrator. For more info on proxy configuration please checkout %url", url: "https://...." }
  }
}

```

where the url is optional